sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Breaking Changes
 * Remove .NET 6.0 support.
```

Comment:
This is not a breaking change; You are targeting `netstandard2.0` so applications still running on .NET 6 will automatically fall back to the standard target.

Reply:
Removed this line.